2015-08-05 00:04:36.301 [info] <0.372.0> Application lager started on node 'core@alex-N550JK'
2015-08-05 00:04:43.337 [warning] <0.371.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-05 00:04:43.338 [info] <0.372.0> Application mnesia exited with reason: stopped
2015-08-05 00:04:43.424 [info] <0.372.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-05 00:09:07.976 [info] <0.372.0> Application lager started on node 'core@alex-N550JK'
2015-08-05 00:09:08.047 [warning] <0.371.0> lager_error_logger_h dropped 53 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-05 00:09:08.048 [info] <0.372.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-05 00:09:15.007 [info] <0.372.0> Application mnesia exited with reason: stopped
2015-08-05 00:09:15.105 [info] <0.372.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-05 01:13:48.204 [info] <0.372.0> Application lager started on node 'core@alex-N550JK'
2015-08-05 01:13:55.258 [warning] <0.371.0> lager_error_logger_h dropped 58 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-05 01:13:55.258 [info] <0.372.0> Application mnesia exited with reason: stopped
2015-08-05 01:13:55.363 [info] <0.372.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-05 01:15:44.654 [info] <0.372.0> Application lager started on node 'core@alex-N550JK'
2015-08-05 01:15:51.710 [warning] <0.371.0> lager_error_logger_h dropped 47 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-05 01:15:51.711 [info] <0.372.0> Application mnesia exited with reason: stopped
2015-08-05 01:15:51.802 [info] <0.372.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-05 01:19:55.922 [info] <0.372.0> Application lager started on node 'core@alex-N550JK'
2015-08-05 01:20:02.951 [warning] <0.371.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-05 01:20:02.951 [info] <0.372.0> Application mnesia exited with reason: stopped
2015-08-05 01:20:03.044 [info] <0.372.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-05 01:21:37.036 [info] <0.372.0> Application lager started on node 'core@alex-N550JK'
2015-08-05 01:21:44.087 [warning] <0.371.0> lager_error_logger_h dropped 60 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-05 01:21:44.087 [info] <0.372.0> Application mnesia exited with reason: stopped
2015-08-05 01:21:44.210 [info] <0.372.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-05 01:38:32.000 [info] <0.372.0> Application lager started on node 'core@alex-N550JK'
2015-08-05 01:38:39.039 [warning] <0.371.0> lager_error_logger_h dropped 14 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-05 01:38:39.039 [info] <0.372.0> Application mnesia exited with reason: stopped
2015-08-05 01:38:39.135 [info] <0.372.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-05 01:42:37.916 [info] <0.372.0> Application lager started on node 'core@alex-N550JK'
2015-08-05 01:42:44.936 [warning] <0.371.0> lager_error_logger_h dropped 58 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-05 01:42:44.936 [info] <0.372.0> Application mnesia exited with reason: stopped
2015-08-05 01:42:45.019 [info] <0.372.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-05 01:46:42.645 [info] <0.372.0> Application lager started on node 'core@alex-N550JK'
2015-08-05 01:46:49.686 [warning] <0.371.0> lager_error_logger_h dropped 57 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-05 01:46:49.686 [info] <0.372.0> Application mnesia exited with reason: stopped
2015-08-05 01:46:49.781 [info] <0.372.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-05 01:52:32.409 [info] <0.372.0> Application lager started on node 'core@alex-N550JK'
2015-08-05 01:52:39.465 [warning] <0.371.0> lager_error_logger_h dropped 61 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-05 01:52:39.465 [info] <0.372.0> Application mnesia exited with reason: stopped
2015-08-05 01:52:39.553 [info] <0.372.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-05 01:53:22.737 [info] <0.372.0> Application lager started on node 'core@alex-N550JK'
2015-08-05 01:53:29.778 [warning] <0.371.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-05 01:53:29.778 [info] <0.372.0> Application mnesia exited with reason: stopped
2015-08-05 01:53:29.893 [info] <0.372.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-05 01:55:24.382 [info] <0.372.0> Application lager started on node 'core@alex-N550JK'
2015-08-05 01:55:31.454 [warning] <0.371.0> lager_error_logger_h dropped 58 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-05 01:55:31.454 [info] <0.372.0> Application mnesia exited with reason: stopped
2015-08-05 01:55:31.551 [info] <0.372.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-06 00:49:29.542 [info] <0.384.0> Application lager started on node 'core@alex-N550JK'
2015-08-06 00:49:36.593 [warning] <0.383.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-06 00:49:36.593 [info] <0.384.0> Application mnesia exited with reason: stopped
2015-08-06 00:49:36.666 [info] <0.384.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-06 01:59:34.292 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-06 01:59:41.362 [warning] <0.384.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-06 01:59:41.362 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-06 01:59:41.444 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-06 02:27:24.912 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-06 02:27:31.953 [warning] <0.384.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-06 02:27:31.953 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-06 02:27:32.025 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-06 02:28:25.815 [error] emulator Error in process <0.753.0> on node 'core@alex-N550JK' with exit value: {{nocatch,[{reason,{error,{invalid_object_member,109}}},{mfa,{ibot_webi_handler,websocket_info,3}},{stacktrace,[{jiffy,encode,2,[{file,"src/jiffy.erl"},{line,41}]},{ibot_webi_handler,websocket_info,3,[{file,"src/ibot_we... 


2015-08-06 02:28:25.817 [error] <0.541.0> Ranch listener http had connection process started with cowboy_protocol:start_link/4 at <0.753.0> exit with reason: {{nocatch,[{reason,{error,{invalid_object_member,109}}},{mfa,{ibot_webi_handler,websocket_info,3}},{stacktrace,[{jiffy,encode,2,[{file,"src/jiffy.erl"},{line,41}]},{ibot_webi_handler,websocket_info,3,[{file,"src/ibot_webi_handler.erl"},{line,149}]},{cowboy_websocket,handler_call,7,[{file,"src/cowboy_websocket.erl"},{line,598}]},{cowboy_protocol,resume,6,[{file,"src/cowboy_protocol.erl"},{line,545}]}]},{msg,{<0.651.0>,{pubsub,wsuiclient},{send_data_to_ui,'BLA_BLA_BLA',{"message #0 for UI"}}}},{req,[{socket,#Port<0.941>},{transport,ranch_tcp},{connection,keepalive},{pid,<0.753.0>},{method,<<"GET">>},{version,'HTTP/1.1'},{peer,{{127,0,0,1},48079}},{host,<<"localhost">>},{host_info,undefined},{port,10101},{path,<<"/websocket">>},{path_info,undefined},{qs,<<>>},{qs_vals,undefined},{bindings,[]},{headers,[{<<"host">>,<<"localhost:10101">>},{<<"connection">>,<<"Upgrade">>},{<<"pragma">>,<<"no-cache">>},{<<"cache-control">>,<<"no-cache">>},{<<"upgrade">>,<<"websocket">>},{<<"origin">>,<<"http://localhost:10101">>},{<<"sec-websocket-version">>,<<"13">>},{<<"user-agent">>,<<"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Ubuntu Chromium/39.0.2171.65 Chrome/39.0.2171.65 Safari/537.36">>},{<<"accept-encoding">>,<<"gzip, deflate, sdch">>},{<<"accept-language">>,<<"en-US,en;q=0.8">>},{<<"sec-websocket-key">>,<<"CW5hgC1fxMh+R5UY8MKQ0g==">>},{<<"sec-websocket-extensions">>,<<"permessage-deflate; client_max_window_bits">>}]},{p_headers,[{<<"sec-websocket-extensions">>,[{<<"permessage-deflate">>,[<<"client_max_window_bits">>]}]},{<<"upgrade">>,[<<"websocket">>]},{<<"connection">>,[<<"upgrade">>]}]},{cookies,undefined},{meta,[{websocket_compress,false},{websocket_version,13}]},{body_state,waiting},{multipart,undefined},{buffer,<<>>},{resp_compress,false},{resp_state,done},{resp_headers,[]},{resp_body,<<>>},{onresponse,undefined}]},{state,undefined_state}]},[{cowboy_protocol,resume,6,[{file,"src/cowboy_protocol.erl"},{line,545}]}]}

2015-08-06 02:32:33.937 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-06 02:32:34.034 [warning] <0.384.0> lager_error_logger_h dropped 58 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-06 02:32:34.035 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-06 02:32:40.980 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-06 02:32:41.085 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-06 02:33:24.399 [error] emulator Error in process <0.753.0> on node 'core@alex-N550JK' with exit value: {[{reason,badarith},{mfa,{ibot_webi_handler,websocket_info,3}},{stacktrace,[{ibot_webi_handler,websocket_info,3,[{file,"src/ibot_webi_handler.erl"},{line,149}]},{cowboy_websocket,handler_call,7,[{file,"src/cowboy_we... 


2015-08-06 02:33:24.400 [error] <0.541.0> Ranch listener http had connection process started with cowboy_protocol:start_link/4 at <0.753.0> exit with reason: {[{reason,badarith},{mfa,{ibot_webi_handler,websocket_info,3}},{stacktrace,[{ibot_webi_handler,websocket_info,3,[{file,"src/ibot_webi_handler.erl"},{line,149}]},{cowboy_websocket,handler_call,7,[{file,"src/cowboy_websocket.erl"},{line,598}]},{cowboy_protocol,resume,6,[{file,"src/cowboy_protocol.erl"},{line,545}]}]},{msg,{<0.651.0>,{pubsub,wsuiclient},{send_data_to_ui,'BLA_BLA_BLA',{"message #0 for UI"}}}},{req,[{socket,#Port<0.941>},{transport,ranch_tcp},{connection,keepalive},{pid,<0.753.0>},{method,<<"GET">>},{version,'HTTP/1.1'},{peer,{{127,0,0,1},48137}},{host,<<"localhost">>},{host_info,undefined},{port,10101},{path,<<"/websocket">>},{path_info,undefined},{qs,<<>>},{qs_vals,undefined},{bindings,[]},{headers,[{<<"host">>,<<"localhost:10101">>},{<<"connection">>,<<"Upgrade">>},{<<"pragma">>,<<"no-cache">>},{<<"cache-control">>,<<"no-cache">>},{<<"upgrade">>,<<"websocket">>},{<<"origin">>,<<"http://localhost:10101">>},{<<"sec-websocket-version">>,<<"13">>},{<<"user-agent">>,<<"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Ubuntu Chromium/39.0.2171.65 Chrome/39.0.2171.65 Safari/537.36">>},{<<"accept-encoding">>,<<"gzip, deflate, sdch">>},{<<"accept-language">>,<<"en-US,en;q=0.8">>},{<<"sec-websocket-key">>,<<"3OPbulz9Hm8fn9XPFslg2w==">>},{<<"sec-websocket-extensions">>,<<"permessage-deflate; client_max_window_bits">>}]},{p_headers,[{<<"sec-websocket-extensions">>,[{<<"permessage-deflate">>,[<<"client_max_window_bits">>]}]},{<<"upgrade">>,[<<"websocket">>]},{<<"connection">>,[<<"upgrade">>]}]},{cookies,undefined},{meta,[{websocket_compress,false},{websocket_version,13}]},{body_state,waiting},{multipart,undefined},{buffer,<<>>},{resp_compress,false},{resp_state,done},{resp_headers,[]},{resp_body,<<>>},{onresponse,undefined}]},{state,undefined_state}],[{cowboy_protocol,resume,6,[{file,"src/cowboy_protocol.erl"},{line,545}]}]}

2015-08-06 02:36:41.604 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-06 02:36:48.645 [warning] <0.384.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-06 02:36:48.646 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-06 02:36:48.724 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-06 02:37:14.383 [error] emulator Error in process <0.754.0> on node 'core@alex-N550JK' with exit value: {{nocatch,[{reason,{error,{invalid_object_member,116}}},{mfa,{ibot_webi_handler,websocket_info,3}},{stacktrace,[{jiffy,encode,2,[{file,"src/jiffy.erl"},{line,41}]},{ibot_webi_handler,websocket_info,3,[{file,"src/ibot_we... 


2015-08-06 02:37:14.384 [error] <0.541.0> Ranch listener http had connection process started with cowboy_protocol:start_link/4 at <0.754.0> exit with reason: {{nocatch,[{reason,{error,{invalid_object_member,116}}},{mfa,{ibot_webi_handler,websocket_info,3}},{stacktrace,[{jiffy,encode,2,[{file,"src/jiffy.erl"},{line,41}]},{ibot_webi_handler,websocket_info,3,[{file,"src/ibot_webi_handler.erl"},{line,149}]},{cowboy_websocket,handler_call,7,[{file,"src/cowboy_websocket.erl"},{line,598}]},{cowboy_protocol,resume,6,[{file,"src/cowboy_protocol.erl"},{line,545}]}]},{msg,{<0.651.0>,{pubsub,wsuiclient},{send_data_to_ui,'BLA_BLA_BLA',{"message #0 for UI"}}}},{req,[{socket,#Port<0.942>},{transport,ranch_tcp},{connection,keepalive},{pid,<0.754.0>},{method,<<"GET">>},{version,'HTTP/1.1'},{peer,{{127,0,0,1},48183}},{host,<<"localhost">>},{host_info,undefined},{port,10101},{path,<<"/websocket">>},{path_info,undefined},{qs,<<>>},{qs_vals,undefined},{bindings,[]},{headers,[{<<"host">>,<<"localhost:10101">>},{<<"connection">>,<<"Upgrade">>},{<<"pragma">>,<<"no-cache">>},{<<"cache-control">>,<<"no-cache">>},{<<"upgrade">>,<<"websocket">>},{<<"origin">>,<<"http://localhost:10101">>},{<<"sec-websocket-version">>,<<"13">>},{<<"user-agent">>,<<"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Ubuntu Chromium/39.0.2171.65 Chrome/39.0.2171.65 Safari/537.36">>},{<<"accept-encoding">>,<<"gzip, deflate, sdch">>},{<<"accept-language">>,<<"en-US,en;q=0.8">>},{<<"sec-websocket-key">>,<<"PR6MxLxOVMZmSUaiDZdfTA==">>},{<<"sec-websocket-extensions">>,<<"permessage-deflate; client_max_window_bits">>}]},{p_headers,[{<<"sec-websocket-extensions">>,[{<<"permessage-deflate">>,[<<"client_max_window_bits">>]}]},{<<"upgrade">>,[<<"websocket">>]},{<<"connection">>,[<<"upgrade">>]}]},{cookies,undefined},{meta,[{websocket_compress,false},{websocket_version,13}]},{body_state,waiting},{multipart,undefined},{buffer,<<>>},{resp_compress,false},{resp_state,done},{resp_headers,[]},{resp_body,<<>>},{onresponse,undefined}]},{state,undefined_state}]},[{cowboy_protocol,resume,6,[{file,"src/cowboy_protocol.erl"},{line,545}]}]}

2015-08-06 02:38:00.491 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-06 02:38:07.563 [warning] <0.384.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-06 02:38:07.563 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-06 02:38:07.648 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-06 02:38:44.601 [error] emulator Error in process <0.756.0> on node 'core@alex-N550JK' with exit value: {{try_clause,ok},[{cowboy_websocket,handler_call,7,[{file,"src/cowboy_websocket.erl"},{line,598}]},{cowboy_protocol,resume,6,[{file,"src/cowboy_protocol.erl"},{line,545}]}]}


2015-08-06 02:38:44.602 [error] <0.542.0> Ranch listener http had connection process started with cowboy_protocol:start_link/4 at <0.756.0> exit with reason: {{try_clause,ok},[{cowboy_websocket,handler_call,7,[{file,"src/cowboy_websocket.erl"},{line,598}]},{cowboy_protocol,resume,6,[{file,"src/cowboy_protocol.erl"},{line,545}]}]}

2015-08-06 02:41:10.680 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-06 02:41:17.705 [warning] <0.384.0> lager_error_logger_h dropped 58 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-06 02:41:17.705 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-06 02:41:17.804 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-07 02:43:10.714 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-07 02:43:17.750 [warning] <0.384.0> lager_error_logger_h dropped 50 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-07 02:43:17.750 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-07 02:43:17.853 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-07 02:54:53.597 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-07 02:55:00.664 [warning] <0.384.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-07 02:55:00.664 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-07 02:55:00.826 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-07 02:56:23.067 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-07 02:56:30.104 [warning] <0.384.0> lager_error_logger_h dropped 58 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-07 02:56:30.104 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-07 02:56:30.210 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-08 23:51:29.018 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-08 23:51:36.049 [warning] <0.384.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-08 23:51:36.049 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-08 23:51:36.154 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-08 23:53:55.219 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-08 23:54:02.215 [warning] <0.384.0> lager_error_logger_h dropped 58 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-08 23:54:02.215 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-08 23:54:02.320 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-09 00:16:38.012 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-09 00:16:45.024 [warning] <0.384.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-09 00:16:45.024 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-09 00:16:45.130 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-09 00:25:09.044 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-09 00:25:16.056 [warning] <0.384.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-09 00:25:16.056 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-09 00:25:16.151 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-09 00:27:30.612 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-09 00:27:37.608 [warning] <0.384.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-09 00:27:37.608 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-09 00:27:37.734 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-09 00:33:17.718 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-09 00:33:24.719 [warning] <0.384.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-09 00:33:24.719 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-09 00:33:24.860 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-10 00:19:11.762 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-10 00:19:18.810 [warning] <0.384.0> lager_error_logger_h dropped 60 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-10 00:19:18.810 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-10 00:19:18.883 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-10 00:28:11.395 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-10 00:28:18.443 [warning] <0.384.0> lager_error_logger_h dropped 58 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-10 00:28:18.443 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-10 00:28:18.537 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-10 01:23:10.180 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-10 01:23:17.190 [warning] <0.384.0> lager_error_logger_h dropped 60 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-10 01:23:17.190 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-10 01:23:17.267 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-10 01:24:58.724 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-10 01:25:05.765 [warning] <0.384.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-10 01:25:05.765 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-10 01:25:05.863 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-10 01:38:03.147 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-10 01:38:10.173 [warning] <0.384.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-10 01:38:10.173 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-10 01:38:10.278 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-10 01:42:59.100 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-10 01:43:06.139 [warning] <0.384.0> lager_error_logger_h dropped 58 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-10 01:43:06.139 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-10 01:43:06.247 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-10 02:06:02.009 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-10 02:06:09.036 [warning] <0.384.0> lager_error_logger_h dropped 60 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-10 02:06:09.036 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-10 02:06:09.109 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-10 02:09:49.943 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-10 02:09:50.006 [warning] <0.384.0> lager_error_logger_h dropped 58 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-10 02:09:50.007 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-10 02:09:56.976 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-10 02:09:57.081 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-10 02:14:43.898 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-10 02:14:50.922 [warning] <0.384.0> lager_error_logger_h dropped 60 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-10 02:14:50.922 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-10 02:14:51.006 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-10 02:24:51.729 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-10 02:24:58.772 [warning] <0.384.0> lager_error_logger_h dropped 58 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-10 02:24:58.772 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-10 02:24:58.853 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-10 02:32:41.796 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-10 02:32:48.837 [warning] <0.384.0> lager_error_logger_h dropped 60 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-10 02:32:48.837 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-10 02:32:48.933 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-10 02:35:26.644 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-10 02:35:33.676 [warning] <0.384.0> lager_error_logger_h dropped 62 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-10 02:35:33.676 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-10 02:35:33.756 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-10 02:37:19.205 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-10 02:37:26.244 [warning] <0.384.0> lager_error_logger_h dropped 60 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-10 02:37:26.244 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-10 02:37:26.315 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-11 02:40:38.776 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-11 02:40:45.832 [warning] <0.384.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-11 02:40:45.832 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-11 02:40:45.903 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-11 02:41:24.979 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-11 02:41:25.043 [warning] <0.384.0> lager_error_logger_h dropped 58 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-11 02:41:25.043 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-11 02:41:32.023 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-11 02:41:32.110 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-11 02:44:49.708 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-11 02:44:56.744 [warning] <0.384.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-11 02:44:56.744 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-11 02:44:56.839 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-11 02:47:04.896 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-11 02:47:11.961 [warning] <0.384.0> lager_error_logger_h dropped 63 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-11 02:47:11.961 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-11 02:47:12.052 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-11 02:51:42.761 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-11 02:52:18.552 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-11 02:52:25.592 [warning] <0.384.0> lager_error_logger_h dropped 60 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-11 02:52:25.592 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-11 02:52:25.676 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-11 02:55:39.287 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-11 02:55:46.326 [warning] <0.384.0> lager_error_logger_h dropped 58 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-11 02:55:46.326 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-11 02:55:46.417 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-11 03:03:18.717 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-11 03:03:25.757 [warning] <0.384.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-11 03:03:25.757 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-11 03:03:25.853 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-13 00:13:07.871 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-13 00:13:08.332 [warning] <0.384.0> lager_error_logger_h dropped 58 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-13 00:13:08.332 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-13 00:13:14.940 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-13 00:13:15.078 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-13 00:32:59.746 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-13 00:33:06.777 [warning] <0.384.0> lager_error_logger_h dropped 63 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-13 00:33:06.777 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-13 00:33:06.881 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-13 00:38:05.418 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-13 00:38:12.458 [warning] <0.384.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-13 00:38:12.458 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-13 00:38:12.539 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-13 00:40:46.802 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-13 00:40:53.833 [warning] <0.384.0> lager_error_logger_h dropped 58 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-13 00:40:53.833 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-13 00:40:53.924 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-13 00:59:27.306 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-13 00:59:34.358 [warning] <0.384.0> lager_error_logger_h dropped 61 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-13 00:59:34.358 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-13 00:59:34.453 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-13 01:00:43.775 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-13 01:00:50.802 [warning] <0.384.0> lager_error_logger_h dropped 57 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-13 01:00:50.802 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-13 01:00:50.896 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-13 01:05:21.219 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-13 01:05:28.248 [warning] <0.384.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-13 01:05:28.248 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-13 01:05:28.352 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-13 01:10:06.069 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-13 01:10:13.099 [warning] <0.384.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-13 01:10:13.099 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-13 01:10:13.193 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-13 01:16:59.566 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-13 01:17:06.598 [warning] <0.384.0> lager_error_logger_h dropped 58 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-13 01:17:06.598 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-13 01:17:06.702 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-13 02:48:21.686 [info] <0.386.0> Application lager started on node 'core@alex-N550JK'
2015-08-13 02:48:28.734 [warning] <0.385.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-13 02:48:28.734 [info] <0.386.0> Application mnesia exited with reason: stopped
2015-08-13 02:48:28.806 [info] <0.386.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-13 03:01:17.158 [info] <0.386.0> Application lager started on node 'core@alex-N550JK'
2015-08-13 03:01:24.955 [warning] <0.385.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-13 03:01:24.955 [info] <0.386.0> Application mnesia exited with reason: stopped
2015-08-13 03:01:25.073 [info] <0.386.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-13 03:04:51.154 [info] <0.386.0> Application lager started on node 'core@alex-N550JK'
2015-08-13 03:04:58.183 [warning] <0.385.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-13 03:04:58.183 [info] <0.386.0> Application mnesia exited with reason: stopped
2015-08-13 03:04:58.265 [info] <0.386.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-13 03:08:25.593 [info] <0.386.0> Application lager started on node 'core@alex-N550JK'
2015-08-13 03:08:32.642 [warning] <0.385.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-13 03:08:32.642 [info] <0.386.0> Application mnesia exited with reason: stopped
2015-08-13 03:08:32.714 [info] <0.386.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-13 03:12:36.727 [info] <0.386.0> Application lager started on node 'core@alex-N550JK'
2015-08-13 03:12:43.777 [warning] <0.385.0> lager_error_logger_h dropped 60 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-13 03:12:43.777 [info] <0.386.0> Application mnesia exited with reason: stopped
2015-08-13 03:12:43.859 [info] <0.386.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-13 03:14:36.308 [info] <0.386.0> Application lager started on node 'core@alex-N550JK'
2015-08-13 03:14:43.361 [warning] <0.385.0> lager_error_logger_h dropped 60 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-13 03:14:43.361 [info] <0.386.0> Application mnesia exited with reason: stopped
2015-08-13 03:14:43.444 [info] <0.386.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-13 03:14:55.766 [info] <0.386.0> Application lager started on node 'core@alex-N550JK'
2015-08-13 03:15:02.806 [warning] <0.385.0> lager_error_logger_h dropped 60 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-13 03:15:02.807 [info] <0.386.0> Application mnesia exited with reason: stopped
2015-08-13 03:15:02.889 [info] <0.386.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-14 01:35:41.648 [info] <0.386.0> Application lager started on node 'core@alex-N550JK'
2015-08-14 01:35:48.947 [warning] <0.385.0> lager_error_logger_h dropped 55 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-14 01:35:48.947 [info] <0.386.0> Application mnesia exited with reason: stopped
2015-08-14 01:35:49.031 [info] <0.386.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-14 01:37:55.859 [info] <0.386.0> Application lager started on node 'core@alex-N550JK'
2015-08-14 01:38:02.904 [warning] <0.385.0> lager_error_logger_h dropped 63 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-14 01:38:02.904 [info] <0.386.0> Application mnesia exited with reason: stopped
2015-08-14 01:38:02.986 [info] <0.386.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-14 01:40:00.391 [info] <0.386.0> Application lager started on node 'core@alex-N550JK'
2015-08-14 01:40:07.442 [warning] <0.385.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-14 01:40:07.442 [info] <0.386.0> Application mnesia exited with reason: stopped
2015-08-14 01:40:07.525 [info] <0.386.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-14 01:44:15.889 [info] <0.386.0> Application lager started on node 'core@alex-N550JK'
2015-08-14 01:44:22.953 [warning] <0.385.0> lager_error_logger_h dropped 47 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-14 01:44:22.953 [info] <0.386.0> Application mnesia exited with reason: stopped
2015-08-14 01:44:23.036 [info] <0.386.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-14 01:50:18.033 [info] <0.386.0> Application lager started on node 'core@alex-N550JK'
2015-08-14 01:50:25.086 [warning] <0.385.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-14 01:50:25.086 [info] <0.386.0> Application mnesia exited with reason: stopped
2015-08-14 01:50:25.171 [info] <0.386.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-14 01:53:28.207 [info] <0.386.0> Application lager started on node 'core@alex-N550JK'
2015-08-14 01:53:35.246 [warning] <0.385.0> lager_error_logger_h dropped 56 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-14 01:53:35.246 [info] <0.386.0> Application mnesia exited with reason: stopped
2015-08-14 01:53:35.328 [info] <0.386.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-14 01:56:52.599 [info] <0.386.0> Application lager started on node 'core@alex-N550JK'
2015-08-14 01:56:59.650 [warning] <0.385.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-14 01:56:59.650 [info] <0.386.0> Application mnesia exited with reason: stopped
2015-08-14 01:56:59.748 [info] <0.386.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-14 02:04:55.989 [info] <0.386.0> Application lager started on node 'core@alex-N550JK'
2015-08-14 02:04:56.060 [warning] <0.385.0> lager_error_logger_h dropped 58 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-14 02:04:56.060 [info] <0.386.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-14 02:05:03.040 [info] <0.386.0> Application mnesia exited with reason: stopped
2015-08-14 02:05:03.133 [info] <0.386.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-14 02:09:05.161 [info] <0.386.0> Application lager started on node 'core@alex-N550JK'
2015-08-14 02:09:12.191 [warning] <0.385.0> lager_error_logger_h dropped 55 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-14 02:09:12.191 [info] <0.386.0> Application mnesia exited with reason: stopped
2015-08-14 02:09:12.287 [info] <0.386.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-14 02:14:24.773 [info] <0.386.0> Application lager started on node 'core@alex-N550JK'
2015-08-14 02:14:31.812 [warning] <0.385.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-14 02:14:31.812 [info] <0.386.0> Application mnesia exited with reason: stopped
2015-08-14 02:14:31.896 [info] <0.386.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-14 02:23:35.200 [info] <0.386.0> Application lager started on node 'core@alex-N550JK'
2015-08-14 02:23:42.253 [warning] <0.385.0> lager_error_logger_h dropped 58 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-14 02:23:42.253 [info] <0.386.0> Application mnesia exited with reason: stopped
2015-08-14 02:23:42.346 [info] <0.386.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-14 02:27:29.165 [info] <0.386.0> Application lager started on node 'core@alex-N550JK'
2015-08-14 02:27:36.192 [warning] <0.385.0> lager_error_logger_h dropped 60 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-14 02:27:36.192 [info] <0.386.0> Application mnesia exited with reason: stopped
2015-08-14 02:27:36.290 [info] <0.386.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-17 01:02:57.068 [info] <0.386.0> Application lager started on node 'core@alex-N550JK'
2015-08-17 01:03:04.100 [warning] <0.385.0> lager_error_logger_h dropped 57 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-17 01:03:04.100 [info] <0.386.0> Application mnesia exited with reason: stopped
2015-08-17 01:03:04.195 [info] <0.386.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-19 01:16:42.699 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-19 01:17:58.076 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-19 01:21:52.513 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-19 01:21:59.554 [warning] <0.384.0> lager_error_logger_h dropped 63 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-19 01:21:59.554 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-19 01:21:59.641 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-19 02:14:54.377 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-19 02:15:01.451 [warning] <0.384.0> lager_error_logger_h dropped 60 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-19 02:15:01.452 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-19 02:15:01.557 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-19 02:15:31.532 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-19 02:15:38.576 [warning] <0.384.0> lager_error_logger_h dropped 63 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-19 02:15:38.576 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-19 02:15:38.670 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-19 02:16:48.644 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-19 02:16:55.701 [warning] <0.384.0> lager_error_logger_h dropped 64 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-19 02:16:55.701 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-19 02:16:55.800 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-19 02:19:17.242 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-19 02:19:24.288 [warning] <0.384.0> lager_error_logger_h dropped 65 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-19 02:19:24.288 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-19 02:19:24.390 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-20 02:06:37.866 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-20 02:06:44.936 [warning] <0.384.0> lager_error_logger_h dropped 63 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-20 02:06:44.936 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-20 02:06:45.024 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-20 02:20:01.541 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-20 02:20:08.610 [warning] <0.384.0> lager_error_logger_h dropped 63 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-20 02:20:08.611 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-20 02:20:08.694 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-20 02:23:32.123 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-20 02:23:39.178 [warning] <0.384.0> lager_error_logger_h dropped 63 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-20 02:23:39.178 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-20 02:23:39.270 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-20 03:09:55.978 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-20 03:09:56.049 [warning] <0.384.0> lager_error_logger_h dropped 57 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-20 03:09:56.050 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-20 03:10:03.000 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-20 03:10:03.075 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-20 03:13:13.060 [error] <0.650.0> gen_server ibot_nodes_srv_connector terminated with reason: {{function_clause,[{gen,call,["test_py_monitor",'$gen_call',{stop},5000],[{file,"gen.erl"},{line,149}]},{gen_server,call,2,[{file,"gen_server.erl"},{line,178}]},{ibot_nodes_srv_connector,handle_info,2,[{file,"src/ibot_nodes_srv_connector.erl"},{line,68}]},{gen_server,try_dispatch,4,[{file,"gen_server.erl"},{line,593}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,659}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,237}]}]},{gen_server,call,[{"test_py_monitor",'core@alex-N550JK'},...]}} in gen_server:call/2 line 182
2015-08-20 03:13:13.060 [error] <0.650.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: {{function_clause,[{gen,call,["test_py_monitor",'$gen_call',{stop},5000],[{file,"gen.erl"},{line,149}]},{gen_server,call,2,[{file,"gen_server.erl"},{line,178}]},{ibot_nodes_srv_connector,handle_info,2,[{file,"src/ibot_nodes_srv_connector.erl"},{line,68}]},{gen_server,try_dispatch,4,[{file,"gen_server.erl"},{line,593}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,659}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,237}]}]},{gen_server,call,[{"test_py_monitor",'core@alex-N550JK'},...]}} in gen_server:terminate/7 line 804
2015-08-20 03:13:13.061 [error] <0.760.0> gen_server test_py_monitor terminated with reason: {{function_clause,[{gen,call,["test_py_monitor",'$gen_call',{stop},5000],[{file,"gen.erl"},{line,149}]},{gen_server,call,2,[{file,"gen_server.erl"},{line,178}]},{ibot_nodes_srv_connector,handle_info,2,[{file,"src/ibot_nodes_srv_connector.erl"},{line,68}]},{gen_server,try_dispatch,4,[{file,"gen_server.erl"},{line,593}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,659}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,237}]}]},{gen_server,call,[{"test_py_monitor",'core@alex-N550JK'},...]}}
2015-08-20 03:13:13.062 [error] <0.760.0> CRASH REPORT Process test_py_monitor with 0 neighbours exited with reason: {{function_clause,[{gen,call,["test_py_monitor",'$gen_call',{stop},5000],[{file,"gen.erl"},{line,149}]},{gen_server,call,2,[{file,"gen_server.erl"},{line,178}]},{ibot_nodes_srv_connector,handle_info,2,[{file,"src/ibot_nodes_srv_connector.erl"},{line,68}]},{gen_server,try_dispatch,4,[{file,"gen_server.erl"},{line,593}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,659}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,237}]}]},{gen_server,call,[{"test_py_monitor",'core@alex-N550JK'},...]}} in gen_server:terminate/7 line 804
2015-08-20 03:13:13.062 [error] <0.763.0> gen_server 'BLA_BLA_BLA_monitor' terminated with reason: {{function_clause,[{gen,call,["test_py_monitor",'$gen_call',{stop},5000],[{file,"gen.erl"},{line,149}]},{gen_server,call,2,[{file,"gen_server.erl"},{line,178}]},{ibot_nodes_srv_connector,handle_info,2,[{file,"src/ibot_nodes_srv_connector.erl"},{line,68}]},{gen_server,try_dispatch,4,[{file,"gen_server.erl"},{line,593}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,659}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,237}]}]},{gen_server,call,[{"test_py_monitor",'core@alex-N550JK'},...]}}
2015-08-20 03:13:13.062 [error] <0.763.0> CRASH REPORT Process 'BLA_BLA_BLA_monitor' with 0 neighbours exited with reason: {{function_clause,[{gen,call,["test_py_monitor",'$gen_call',{stop},5000],[{file,"gen.erl"},{line,149}]},{gen_server,call,2,[{file,"gen_server.erl"},{line,178}]},{ibot_nodes_srv_connector,handle_info,2,[{file,"src/ibot_nodes_srv_connector.erl"},{line,68}]},{gen_server,try_dispatch,4,[{file,"gen_server.erl"},{line,593}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,659}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,237}]}]},{gen_server,call,[{"test_py_monitor",'core@alex-N550JK'},...]}} in gen_server:terminate/7 line 804
2015-08-20 03:13:13.062 [error] <0.648.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.650.0> exit with reason {{function_clause,[{gen,call,["test_py_monitor",'$gen_call',{stop},5000],[{file,"gen.erl"},{line,149}]},{gen_server,call,2,[{file,"gen_server.erl"},{line,178}]},{ibot_nodes_srv_connector,handle_info,2,[{file,"src/ibot_nodes_srv_connector.erl"},{line,68}]},{gen_server,try_dispatch,4,[{file,"gen_server.erl"},{line,593}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,659}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,237}]}]},{gen_server,call,[{"test_py_monitor",'core@alex-N550JK'},...]}} in context child_terminated
2015-08-20 03:13:15.285 [error] <0.784.0> gen_server ibot_nodes_srv_connector terminated with reason: {{function_clause,[{gen,call,["BLA_BLA_BLA_monitor",'$gen_call',{stop},5000],[{file,"gen.erl"},{line,149}]},{gen_server,call,2,[{file,"gen_server.erl"},{line,178}]},{ibot_nodes_srv_connector,handle_info,2,[{file,"src/ibot_nodes_srv_connector.erl"},{line,68}]},{gen_server,try_dispatch,4,[{file,"gen_server.erl"},{line,593}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,659}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,237}]}]},{gen_server,call,[{"BLA_BLA_BLA_monitor",...},...]}} in gen_server:call/2 line 182
2015-08-20 03:13:15.286 [error] <0.784.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: {{function_clause,[{gen,call,["BLA_BLA_BLA_monitor",'$gen_call',{stop},5000],[{file,"gen.erl"},{line,149}]},{gen_server,call,2,[{file,"gen_server.erl"},{line,178}]},{ibot_nodes_srv_connector,handle_info,2,[{file,"src/ibot_nodes_srv_connector.erl"},{line,68}]},{gen_server,try_dispatch,4,[{file,"gen_server.erl"},{line,593}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,659}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,237}]}]},{gen_server,call,[{"BLA_BLA_BLA_monitor",...},...]}} in gen_server:terminate/7 line 804
2015-08-20 03:13:15.287 [error] <0.648.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.784.0> exit with reason {{function_clause,[{gen,call,["BLA_BLA_BLA_monitor",'$gen_call',{stop},5000],[{file,"gen.erl"},{line,149}]},{gen_server,call,2,[{file,"gen_server.erl"},{line,178}]},{ibot_nodes_srv_connector,handle_info,2,[{file,"src/ibot_nodes_srv_connector.erl"},{line,68}]},{gen_server,try_dispatch,4,[{file,"gen_server.erl"},{line,593}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,659}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,237}]}]},{gen_server,call,[{"BLA_BLA_BLA_monitor",...},...]}} in context child_terminated
2015-08-20 03:13:46.530 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-20 03:13:53.549 [warning] <0.384.0> lager_error_logger_h dropped 57 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-20 03:13:53.550 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-20 03:13:53.659 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-20 03:18:58.016 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-20 03:19:05.048 [warning] <0.384.0> lager_error_logger_h dropped 54 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-20 03:19:05.048 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-20 03:19:05.163 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-22 01:34:08.152 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-22 01:34:15.204 [warning] <0.384.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-22 01:34:15.204 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-22 01:34:15.285 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-22 01:34:48.314 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-22 01:34:55.366 [warning] <0.384.0> lager_error_logger_h dropped 58 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-22 01:34:55.366 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-22 01:34:55.450 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-25 01:39:11.504 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-25 01:39:18.523 [warning] <0.384.0> lager_error_logger_h dropped 64 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-25 01:39:18.523 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-25 01:39:18.619 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-25 01:40:30.294 [error] <0.647.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.649.0> exit with reason error in context child_terminated
2015-08-25 01:41:51.888 [error] <0.647.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.752.0> exit with reason error in context child_terminated
2015-08-25 01:51:55.845 [error] <0.757.0> gen_server ibot_nodes_srv_connector terminated with reason: {{function_clause,[{gen,call,["test_py_monitor",'$gen_call',{stop},5000],[{file,"gen.erl"},{line,149}]},{gen_server,call,2,[{file,"gen_server.erl"},{line,178}]},{ibot_nodes_srv_connector,handle_info,2,[{file,"src/ibot_nodes_srv_connector.erl"},{line,68}]},{gen_server,try_dispatch,4,[{file,"gen_server.erl"},{line,593}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,659}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,237}]}]},{gen_server,call,[{"test_py_monitor",'core@alex-N550JK'},...]}} in gen_server:call/2 line 182
2015-08-25 01:51:55.845 [error] <0.757.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: {{function_clause,[{gen,call,["test_py_monitor",'$gen_call',{stop},5000],[{file,"gen.erl"},{line,149}]},{gen_server,call,2,[{file,"gen_server.erl"},{line,178}]},{ibot_nodes_srv_connector,handle_info,2,[{file,"src/ibot_nodes_srv_connector.erl"},{line,68}]},{gen_server,try_dispatch,4,[{file,"gen_server.erl"},{line,593}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,659}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,237}]}]},{gen_server,call,[{"test_py_monitor",'core@alex-N550JK'},...]}} in gen_server:terminate/7 line 804
2015-08-25 01:51:55.845 [error] <0.779.0> gen_server test_py_monitor terminated with reason: {{function_clause,[{gen,call,["test_py_monitor",'$gen_call',{stop},5000],[{file,"gen.erl"},{line,149}]},{gen_server,call,2,[{file,"gen_server.erl"},{line,178}]},{ibot_nodes_srv_connector,handle_info,2,[{file,"src/ibot_nodes_srv_connector.erl"},{line,68}]},{gen_server,try_dispatch,4,[{file,"gen_server.erl"},{line,593}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,659}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,237}]}]},{gen_server,call,[{"test_py_monitor",'core@alex-N550JK'},...]}}
2015-08-25 01:51:55.846 [error] <0.779.0> CRASH REPORT Process test_py_monitor with 0 neighbours exited with reason: {{function_clause,[{gen,call,["test_py_monitor",'$gen_call',{stop},5000],[{file,"gen.erl"},{line,149}]},{gen_server,call,2,[{file,"gen_server.erl"},{line,178}]},{ibot_nodes_srv_connector,handle_info,2,[{file,"src/ibot_nodes_srv_connector.erl"},{line,68}]},{gen_server,try_dispatch,4,[{file,"gen_server.erl"},{line,593}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,659}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,237}]}]},{gen_server,call,[{"test_py_monitor",'core@alex-N550JK'},...]}} in gen_server:terminate/7 line 804
2015-08-25 01:51:55.846 [error] <0.782.0> gen_server 'BLA_BLA_BLA_monitor' terminated with reason: {{function_clause,[{gen,call,["test_py_monitor",'$gen_call',{stop},5000],[{file,"gen.erl"},{line,149}]},{gen_server,call,2,[{file,"gen_server.erl"},{line,178}]},{ibot_nodes_srv_connector,handle_info,2,[{file,"src/ibot_nodes_srv_connector.erl"},{line,68}]},{gen_server,try_dispatch,4,[{file,"gen_server.erl"},{line,593}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,659}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,237}]}]},{gen_server,call,[{"test_py_monitor",'core@alex-N550JK'},...]}}
2015-08-25 01:51:55.847 [error] <0.647.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.757.0> exit with reason {{function_clause,[{gen,call,["test_py_monitor",'$gen_call',{stop},5000],[{file,"gen.erl"},{line,149}]},{gen_server,call,2,[{file,"gen_server.erl"},{line,178}]},{ibot_nodes_srv_connector,handle_info,2,[{file,"src/ibot_nodes_srv_connector.erl"},{line,68}]},{gen_server,try_dispatch,4,[{file,"gen_server.erl"},{line,593}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,659}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,237}]}]},{gen_server,call,[{"test_py_monitor",'core@alex-N550JK'},...]}} in context child_terminated
2015-08-25 01:51:55.847 [error] <0.782.0> CRASH REPORT Process 'BLA_BLA_BLA_monitor' with 0 neighbours exited with reason: {{function_clause,[{gen,call,["test_py_monitor",'$gen_call',{stop},5000],[{file,"gen.erl"},{line,149}]},{gen_server,call,2,[{file,"gen_server.erl"},{line,178}]},{ibot_nodes_srv_connector,handle_info,2,[{file,"src/ibot_nodes_srv_connector.erl"},{line,68}]},{gen_server,try_dispatch,4,[{file,"gen_server.erl"},{line,593}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,659}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,237}]}]},{gen_server,call,[{"test_py_monitor",'core@alex-N550JK'},...]}} in gen_server:terminate/7 line 804
2015-08-25 01:52:42.559 [error] <0.808.0> gen_server ibot_nodes_srv_connector terminated with reason: {{function_clause,[{gen,call,["BLA_BLA_BLA_monitor",'$gen_call',{stop},5000],[{file,"gen.erl"},{line,149}]},{gen_server,call,2,[{file,"gen_server.erl"},{line,178}]},{ibot_nodes_srv_connector,handle_info,2,[{file,"src/ibot_nodes_srv_connector.erl"},{line,68}]},{gen_server,try_dispatch,4,[{file,"gen_server.erl"},{line,593}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,659}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,237}]}]},{gen_server,call,[{"BLA_BLA_BLA_monitor",...},...]}} in gen_server:call/2 line 182
2015-08-25 01:52:42.560 [error] <0.808.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: {{function_clause,[{gen,call,["BLA_BLA_BLA_monitor",'$gen_call',{stop},5000],[{file,"gen.erl"},{line,149}]},{gen_server,call,2,[{file,"gen_server.erl"},{line,178}]},{ibot_nodes_srv_connector,handle_info,2,[{file,"src/ibot_nodes_srv_connector.erl"},{line,68}]},{gen_server,try_dispatch,4,[{file,"gen_server.erl"},{line,593}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,659}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,237}]}]},{gen_server,call,[{"BLA_BLA_BLA_monitor",...},...]}} in gen_server:terminate/7 line 804
2015-08-25 01:52:42.561 [error] <0.647.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.808.0> exit with reason {{function_clause,[{gen,call,["BLA_BLA_BLA_monitor",'$gen_call',{stop},5000],[{file,"gen.erl"},{line,149}]},{gen_server,call,2,[{file,"gen_server.erl"},{line,178}]},{ibot_nodes_srv_connector,handle_info,2,[{file,"src/ibot_nodes_srv_connector.erl"},{line,68}]},{gen_server,try_dispatch,4,[{file,"gen_server.erl"},{line,593}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,659}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,237}]}]},{gen_server,call,[{"BLA_BLA_BLA_monitor",...},...]}} in context child_terminated
2015-08-25 01:52:42.561 [error] <0.812.0> gen_server test_py_monitor terminated with reason: {{function_clause,[{gen,call,["BLA_BLA_BLA_monitor",'$gen_call',{stop},5000],[{file,"gen.erl"},{line,149}]},{gen_server,call,2,[{file,"gen_server.erl"},{line,178}]},{ibot_nodes_srv_connector,handle_info,2,[{file,"src/ibot_nodes_srv_connector.erl"},{line,68}]},{gen_server,try_dispatch,4,[{file,"gen_server.erl"},{line,593}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,659}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,237}]}]},{gen_server,call,[{"BLA_BLA_BLA_monitor",...},...]}}
2015-08-25 01:52:42.562 [error] <0.812.0> CRASH REPORT Process test_py_monitor with 0 neighbours exited with reason: {{function_clause,[{gen,call,["BLA_BLA_BLA_monitor",'$gen_call',{stop},5000],[{file,"gen.erl"},{line,149}]},{gen_server,call,2,[{file,"gen_server.erl"},{line,178}]},{ibot_nodes_srv_connector,handle_info,2,[{file,"src/ibot_nodes_srv_connector.erl"},{line,68}]},{gen_server,try_dispatch,4,[{file,"gen_server.erl"},{line,593}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,659}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,237}]}]},{gen_server,call,[{"BLA_BLA_BLA_monitor",...},...]}} in gen_server:terminate/7 line 804
2015-08-25 01:55:42.319 [error] <0.813.0> gen_server ibot_nodes_srv_connector terminated with reason: {{function_clause,[{gen,call,["BLA_BLA_BLA_monitor",'$gen_call',{stop},5000],[{file,"gen.erl"},{line,149}]},{gen_server,call,2,[{file,"gen_server.erl"},{line,178}]},{ibot_nodes_srv_connector,handle_info,2,[{file,"src/ibot_nodes_srv_connector.erl"},{line,68}]},{gen_server,try_dispatch,4,[{file,"gen_server.erl"},{line,593}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,659}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,237}]}]},{gen_server,call,[{"BLA_BLA_BLA_monitor",...},...]}} in gen_server:call/2 line 182
2015-08-25 01:55:42.319 [error] <0.813.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: {{function_clause,[{gen,call,["BLA_BLA_BLA_monitor",'$gen_call',{stop},5000],[{file,"gen.erl"},{line,149}]},{gen_server,call,2,[{file,"gen_server.erl"},{line,178}]},{ibot_nodes_srv_connector,handle_info,2,[{file,"src/ibot_nodes_srv_connector.erl"},{line,68}]},{gen_server,try_dispatch,4,[{file,"gen_server.erl"},{line,593}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,659}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,237}]}]},{gen_server,call,[{"BLA_BLA_BLA_monitor",...},...]}} in gen_server:terminate/7 line 804
2015-08-25 01:55:42.320 [error] <0.647.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.813.0> exit with reason {{function_clause,[{gen,call,["BLA_BLA_BLA_monitor",'$gen_call',{stop},5000],[{file,"gen.erl"},{line,149}]},{gen_server,call,2,[{file,"gen_server.erl"},{line,178}]},{ibot_nodes_srv_connector,handle_info,2,[{file,"src/ibot_nodes_srv_connector.erl"},{line,68}]},{gen_server,try_dispatch,4,[{file,"gen_server.erl"},{line,593}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,659}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,237}]}]},{gen_server,call,[{"BLA_BLA_BLA_monitor",...},...]}} in context child_terminated
2015-08-25 01:55:42.321 [error] <0.816.0> gen_server 'BLA_BLA_BLA_monitor' terminated with reason: {{function_clause,[{gen,call,["BLA_BLA_BLA_monitor",'$gen_call',{stop},5000],[{file,"gen.erl"},{line,149}]},{gen_server,call,2,[{file,"gen_server.erl"},{line,178}]},{ibot_nodes_srv_connector,handle_info,2,[{file,"src/ibot_nodes_srv_connector.erl"},{line,68}]},{gen_server,try_dispatch,4,[{file,"gen_server.erl"},{line,593}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,659}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,237}]}]},{gen_server,call,[{"BLA_BLA_BLA_monitor",...},...]}}
2015-08-25 01:55:42.321 [error] <0.816.0> CRASH REPORT Process 'BLA_BLA_BLA_monitor' with 0 neighbours exited with reason: {{function_clause,[{gen,call,["BLA_BLA_BLA_monitor",'$gen_call',{stop},5000],[{file,"gen.erl"},{line,149}]},{gen_server,call,2,[{file,"gen_server.erl"},{line,178}]},{ibot_nodes_srv_connector,handle_info,2,[{file,"src/ibot_nodes_srv_connector.erl"},{line,68}]},{gen_server,try_dispatch,4,[{file,"gen_server.erl"},{line,593}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,659}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,237}]}]},{gen_server,call,[{"BLA_BLA_BLA_monitor",...},...]}} in gen_server:terminate/7 line 804
2015-08-25 01:56:08.099 [error] <0.832.0> gen_server ibot_nodes_srv_connector terminated with reason: {{function_clause,[{gen,call,["test_py_monitor",'$gen_call',{stop},5000],[{file,"gen.erl"},{line,149}]},{gen_server,call,2,[{file,"gen_server.erl"},{line,178}]},{ibot_nodes_srv_connector,handle_info,2,[{file,"src/ibot_nodes_srv_connector.erl"},{line,68}]},{gen_server,try_dispatch,4,[{file,"gen_server.erl"},{line,593}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,659}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,237}]}]},{gen_server,call,[{"test_py_monitor",'core@alex-N550JK'},...]}} in gen_server:call/2 line 182
2015-08-25 01:56:08.099 [error] <0.832.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: {{function_clause,[{gen,call,["test_py_monitor",'$gen_call',{stop},5000],[{file,"gen.erl"},{line,149}]},{gen_server,call,2,[{file,"gen_server.erl"},{line,178}]},{ibot_nodes_srv_connector,handle_info,2,[{file,"src/ibot_nodes_srv_connector.erl"},{line,68}]},{gen_server,try_dispatch,4,[{file,"gen_server.erl"},{line,593}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,659}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,237}]}]},{gen_server,call,[{"test_py_monitor",'core@alex-N550JK'},...]}} in gen_server:terminate/7 line 804
2015-08-25 01:56:08.099 [error] <0.647.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.832.0> exit with reason {{function_clause,[{gen,call,["test_py_monitor",'$gen_call',{stop},5000],[{file,"gen.erl"},{line,149}]},{gen_server,call,2,[{file,"gen_server.erl"},{line,178}]},{ibot_nodes_srv_connector,handle_info,2,[{file,"src/ibot_nodes_srv_connector.erl"},{line,68}]},{gen_server,try_dispatch,4,[{file,"gen_server.erl"},{line,593}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,659}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,237}]}]},{gen_server,call,[{"test_py_monitor",'core@alex-N550JK'},...]}} in context child_terminated
2015-08-25 01:57:54.509 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-25 01:58:01.553 [warning] <0.384.0> lager_error_logger_h dropped 58 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-25 01:58:01.553 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-25 01:58:01.635 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-25 02:12:41.273 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-25 02:12:48.321 [warning] <0.384.0> lager_error_logger_h dropped 60 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-25 02:12:48.321 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-25 02:12:48.404 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-25 02:16:42.608 [error] <0.809.0> gen_server 'BLA_BLA_BLA_monitor' terminated with reason: test
2015-08-25 02:16:42.609 [error] <0.806.0> gen_server test_py_monitor terminated with reason: test
2015-08-25 02:16:42.609 [error] <0.806.0> CRASH REPORT Process test_py_monitor with 0 neighbours exited with reason: test in gen_server:terminate/7 line 804
2015-08-25 02:16:42.609 [error] <0.809.0> CRASH REPORT Process 'BLA_BLA_BLA_monitor' with 0 neighbours exited with reason: test in gen_server:terminate/7 line 804
2015-08-25 02:16:42.609 [error] <0.647.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.649.0> exit with reason test in context child_terminated
2015-08-30 02:00:31.527 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-30 02:00:38.582 [warning] <0.384.0> lager_error_logger_h dropped 56 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-30 02:00:38.582 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-30 02:00:38.662 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-30 17:17:57.230 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-30 17:18:04.301 [warning] <0.384.0> lager_error_logger_h dropped 63 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-30 17:18:04.301 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-30 17:18:04.382 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-30 17:26:46.495 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-30 17:26:53.526 [warning] <0.384.0> lager_error_logger_h dropped 64 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-30 17:26:53.526 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-30 17:26:53.617 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-30 17:27:30.669 [error] <0.647.0> Supervisor ibot_nodes_sup had child 'BLA_BLA_BLA_monitor' started with ibot_nodes_srv_monitor:start_link({"BLA_BLA_BLA",'BLA_BLA_BLA','alex-N550JK','BLA_BLA_BLA@alex-N550JK'}) at <0.756.0> exit with reason normal in context child_terminated
2015-08-30 17:27:30.697 [error] <0.647.0> Supervisor ibot_nodes_sup had child 'BLA_BLA_BLA_monitor' started with ibot_nodes_srv_monitor:start_link({"BLA_BLA_BLA",'BLA_BLA_BLA','alex-N550JK','BLA_BLA_BLA@alex-N550JK'}) at <0.757.0> exit with reason normal in context child_terminated
2015-08-30 17:27:30.701 [error] <0.647.0> Supervisor ibot_nodes_sup had child 'BLA_BLA_BLA_monitor' started with ibot_nodes_srv_monitor:start_link({"BLA_BLA_BLA",'BLA_BLA_BLA','alex-N550JK','BLA_BLA_BLA@alex-N550JK'}) at <0.760.0> exit with reason normal in context child_terminated
2015-08-30 17:27:30.709 [error] <0.647.0> Supervisor ibot_nodes_sup had child 'BLA_BLA_BLA_monitor' started with ibot_nodes_srv_monitor:start_link({"BLA_BLA_BLA",'BLA_BLA_BLA','alex-N550JK','BLA_BLA_BLA@alex-N550JK'}) at <0.763.0> exit with reason normal in context child_terminated
2015-08-30 17:27:30.711 [error] <0.647.0> Supervisor ibot_nodes_sup had child 'BLA_BLA_BLA_monitor' started with ibot_nodes_srv_monitor:start_link({"BLA_BLA_BLA",'BLA_BLA_BLA','alex-N550JK','BLA_BLA_BLA@alex-N550JK'}) at <0.766.0> exit with reason normal in context child_terminated
2015-08-30 17:27:30.714 [error] <0.647.0> Supervisor ibot_nodes_sup had child 'BLA_BLA_BLA_monitor' started with ibot_nodes_srv_monitor:start_link({"BLA_BLA_BLA",'BLA_BLA_BLA','alex-N550JK','BLA_BLA_BLA@alex-N550JK'}) at <0.769.0> exit with reason normal in context child_terminated
2015-08-30 17:27:30.714 [error] <0.647.0> Supervisor ibot_nodes_sup had child 'BLA_BLA_BLA_monitor' started with ibot_nodes_srv_monitor:start_link({"BLA_BLA_BLA",'BLA_BLA_BLA','alex-N550JK','BLA_BLA_BLA@alex-N550JK'}) at <0.769.0> exit with reason reached_max_restart_intensity in context shutdown
2015-08-30 17:27:30.716 [info] <0.385.0> Application ibot_nodes exited with reason: shutdown
2015-08-30 17:28:22.764 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-30 17:28:29.797 [warning] <0.384.0> lager_error_logger_h dropped 58 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-30 17:28:29.797 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-30 17:28:29.893 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-30 17:36:34.537 [error] <0.782.0> gen_server 'BLA_BLA_BLA_monitor' terminated with reason: error
2015-08-30 17:36:34.538 [error] <0.648.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.650.0> exit with reason error in context child_terminated
2015-08-30 17:36:34.538 [error] <0.782.0> CRASH REPORT Process 'BLA_BLA_BLA_monitor' with 0 neighbours exited with reason: error in gen_server:terminate/7 line 804
2015-08-30 17:37:06.517 [error] <0.792.0> gen_server ibot_nodes_srv_connector terminated with reason: no such process or port in call to gen_server:call('BLA_BLA_BLA_monitor', stop) in gen_server:call/2 line 182
2015-08-30 17:37:06.517 [error] <0.792.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: no such process or port in call to gen_server:call('BLA_BLA_BLA_monitor', stop) in gen_server:terminate/7 line 804
2015-08-30 17:37:06.517 [error] <0.798.0> gen_server test_py_monitor terminated with reason: no such process or port in call to gen_server:call('BLA_BLA_BLA_monitor', stop)
2015-08-30 17:37:06.518 [error] <0.798.0> CRASH REPORT Process test_py_monitor with 0 neighbours exited with reason: no such process or port in call to gen_server:call('BLA_BLA_BLA_monitor', stop) in gen_server:terminate/7 line 804
2015-08-30 17:37:06.518 [error] <0.648.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.792.0> exit with reason no such process or port in call to gen_server:call('BLA_BLA_BLA_monitor', stop) in context child_terminated
2015-08-30 17:37:40.870 [error] <0.800.0> gen_server ibot_nodes_srv_connector terminated with reason: no such process or port in call to gen_server:call('BLA_BLA_BLA_monitor', stop) in gen_server:call/2 line 182
2015-08-30 17:37:40.870 [error] <0.800.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: no such process or port in call to gen_server:call('BLA_BLA_BLA_monitor', stop) in gen_server:terminate/7 line 804
2015-08-30 17:37:40.871 [error] <0.648.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.800.0> exit with reason no such process or port in call to gen_server:call('BLA_BLA_BLA_monitor', stop) in context child_terminated
2015-08-30 17:41:05.446 [error] <0.818.0> gen_server ibot_nodes_srv_connector terminated with reason: no such process or port in call to gen_server:call(test_py_monitor, stop) in gen_server:call/2 line 182
2015-08-30 17:41:05.447 [error] <0.818.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: no such process or port in call to gen_server:call(test_py_monitor, stop) in gen_server:terminate/7 line 804
2015-08-30 17:41:05.447 [error] <0.648.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.818.0> exit with reason no such process or port in call to gen_server:call(test_py_monitor, stop) in context child_terminated
2015-08-30 17:42:27.984 [error] <0.648.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.836.0> exit with reason error in context child_terminated
2015-08-30 17:42:27.984 [error] <0.872.0> gen_server 'BLA_BLA_BLA_monitor' terminated with reason: error
2015-08-30 17:42:27.984 [error] <0.868.0> gen_server test_py_monitor terminated with reason: error
2015-08-30 17:42:27.984 [error] <0.872.0> CRASH REPORT Process 'BLA_BLA_BLA_monitor' with 0 neighbours exited with reason: error in gen_server:terminate/7 line 804
2015-08-30 17:42:27.984 [error] <0.868.0> CRASH REPORT Process test_py_monitor with 0 neighbours exited with reason: error in gen_server:terminate/7 line 804
2015-08-30 17:42:33.095 [error] <0.648.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.873.0> exit with reason error in context child_terminated
2015-08-30 17:42:43.987 [error] <0.885.0> gen_server ibot_nodes_srv_connector terminated with reason: no such process or port in call to gen_server:call('BLA_BLA_BLA_monitor', stop) in gen_server:call/2 line 182
2015-08-30 17:42:43.988 [error] <0.885.0> CRASH REPORT Process ibot_nodes_srv_connector with 0 neighbours exited with reason: no such process or port in call to gen_server:call('BLA_BLA_BLA_monitor', stop) in gen_server:terminate/7 line 804
2015-08-30 17:42:43.988 [error] <0.648.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.885.0> exit with reason no such process or port in call to gen_server:call('BLA_BLA_BLA_monitor', stop) in context child_terminated
2015-08-30 17:44:05.128 [error] <0.892.0> gen_server 'BLA_BLA_BLA_monitor' terminated with reason: error
2015-08-30 17:44:05.129 [error] <0.892.0> CRASH REPORT Process 'BLA_BLA_BLA_monitor' with 0 neighbours exited with reason: error in gen_server:terminate/7 line 804
2015-08-30 17:44:05.129 [error] <0.648.0> Supervisor ibot_nodes_sup had child ibot_nodes_srv_connector started with ibot_nodes_srv_connector:start_link() at <0.888.0> exit with reason error in context child_terminated
2015-08-30 17:55:19.609 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-30 17:55:26.669 [warning] <0.384.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-30 17:55:26.669 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-30 17:55:26.781 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-30 17:56:00.739 [error] <0.647.0> Supervisor ibot_nodes_sup had child 'BLA_BLA_BLA_monitor' started with ibot_nodes_srv_monitor:start_link({"BLA_BLA_BLA",'BLA_BLA_BLA','alex-N550JK','BLA_BLA_BLA@alex-N550JK'}) at <0.761.0> exit with reason normal in context child_terminated
2015-08-30 17:56:28.076 [error] <0.647.0> Supervisor ibot_nodes_sup had child 'BLA_BLA_BLA_monitor' started with ibot_nodes_srv_monitor:start_link({"BLA_BLA_BLA",'BLA_BLA_BLA','alex-N550JK','BLA_BLA_BLA@alex-N550JK'}) at <0.776.0> exit with reason normal in context child_terminated
2015-08-30 17:56:28.081 [error] <0.647.0> Supervisor ibot_nodes_sup had child 'BLA_BLA_BLA_monitor' started with ibot_nodes_srv_monitor:start_link({"BLA_BLA_BLA",'BLA_BLA_BLA','alex-N550JK','BLA_BLA_BLA@alex-N550JK'}) at <0.788.0> exit with reason normal in context child_terminated
2015-08-30 17:56:28.083 [error] <0.647.0> Supervisor ibot_nodes_sup had child 'BLA_BLA_BLA_monitor' started with ibot_nodes_srv_monitor:start_link({"BLA_BLA_BLA",'BLA_BLA_BLA','alex-N550JK','BLA_BLA_BLA@alex-N550JK'}) at <0.791.0> exit with reason normal in context child_terminated
2015-08-30 17:56:28.085 [error] <0.647.0> Supervisor ibot_nodes_sup had child 'BLA_BLA_BLA_monitor' started with ibot_nodes_srv_monitor:start_link({"BLA_BLA_BLA",'BLA_BLA_BLA','alex-N550JK','BLA_BLA_BLA@alex-N550JK'}) at <0.794.0> exit with reason normal in context child_terminated
2015-08-30 17:56:28.103 [error] <0.647.0> Supervisor ibot_nodes_sup had child 'BLA_BLA_BLA_monitor' started with ibot_nodes_srv_monitor:start_link({"BLA_BLA_BLA",'BLA_BLA_BLA','alex-N550JK','BLA_BLA_BLA@alex-N550JK'}) at <0.797.0> exit with reason normal in context child_terminated
2015-08-30 17:56:28.111 [error] <0.647.0> Supervisor ibot_nodes_sup had child 'BLA_BLA_BLA_monitor' started with ibot_nodes_srv_monitor:start_link({"BLA_BLA_BLA",'BLA_BLA_BLA','alex-N550JK','BLA_BLA_BLA@alex-N550JK'}) at <0.800.0> exit with reason normal in context child_terminated
2015-08-30 17:56:28.112 [error] <0.647.0> Supervisor ibot_nodes_sup had child 'BLA_BLA_BLA_monitor' started with ibot_nodes_srv_monitor:start_link({"BLA_BLA_BLA",'BLA_BLA_BLA','alex-N550JK','BLA_BLA_BLA@alex-N550JK'}) at <0.800.0> exit with reason reached_max_restart_intensity in context shutdown
2015-08-30 17:56:28.112 [info] <0.385.0> Application ibot_nodes exited with reason: shutdown
2015-08-30 17:58:14.125 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-30 17:58:21.187 [warning] <0.384.0> lager_error_logger_h dropped 58 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-30 17:58:21.187 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-30 17:58:21.272 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-30 17:58:54.193 [error] <0.648.0> Supervisor ibot_nodes_sup had child 'BLA_BLA_BLA_monitor' started with ibot_nodes_srv_monitor:start_link({"BLA_BLA_BLA",'BLA_BLA_BLA','alex-N550JK','BLA_BLA_BLA@alex-N550JK'}) at <0.757.0> exit with reason normal in context child_terminated
2015-08-30 18:00:04.348 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-30 18:00:11.387 [warning] <0.384.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-30 18:00:11.387 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-30 18:00:11.458 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-31 00:10:19.361 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-31 00:10:26.410 [warning] <0.384.0> lager_error_logger_h dropped 61 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-31 00:10:26.410 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-31 00:10:26.507 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-31 00:14:26.280 [error] <0.648.0> Supervisor ibot_nodes_sup had child 'BLA_BLA_BLA_monitor' started with ibot_nodes_srv_monitor:start_link({"BLA_BLA_BLA",'BLA_BLA_BLA','alex-N550JK','BLA_BLA_BLA@alex-N550JK'}) at <0.764.0> exit with reason normal in context child_terminated
2015-08-31 00:14:26.284 [error] <0.648.0> Supervisor ibot_nodes_sup had child 'BLA_BLA_BLA_monitor' started with ibot_nodes_srv_monitor:start_link({"BLA_BLA_BLA",'BLA_BLA_BLA','alex-N550JK','BLA_BLA_BLA@alex-N550JK'}) at <0.766.0> exit with reason normal in context child_terminated
2015-08-31 00:14:26.285 [error] <0.648.0> Supervisor ibot_nodes_sup had child 'BLA_BLA_BLA_monitor' started with ibot_nodes_srv_monitor:start_link({"BLA_BLA_BLA",'BLA_BLA_BLA','alex-N550JK','BLA_BLA_BLA@alex-N550JK'}) at <0.769.0> exit with reason normal in context child_terminated
2015-08-31 00:14:26.290 [error] <0.648.0> Supervisor ibot_nodes_sup had child 'BLA_BLA_BLA_monitor' started with ibot_nodes_srv_monitor:start_link({"BLA_BLA_BLA",'BLA_BLA_BLA','alex-N550JK','BLA_BLA_BLA@alex-N550JK'}) at <0.772.0> exit with reason normal in context child_terminated
2015-08-31 00:14:26.294 [error] <0.648.0> Supervisor ibot_nodes_sup had child 'BLA_BLA_BLA_monitor' started with ibot_nodes_srv_monitor:start_link({"BLA_BLA_BLA",'BLA_BLA_BLA','alex-N550JK','BLA_BLA_BLA@alex-N550JK'}) at <0.775.0> exit with reason normal in context child_terminated
2015-08-31 00:14:26.297 [error] <0.648.0> Supervisor ibot_nodes_sup had child 'BLA_BLA_BLA_monitor' started with ibot_nodes_srv_monitor:start_link({"BLA_BLA_BLA",'BLA_BLA_BLA','alex-N550JK','BLA_BLA_BLA@alex-N550JK'}) at <0.778.0> exit with reason normal in context child_terminated
2015-08-31 00:14:26.297 [error] <0.648.0> Supervisor ibot_nodes_sup had child 'BLA_BLA_BLA_monitor' started with ibot_nodes_srv_monitor:start_link({"BLA_BLA_BLA",'BLA_BLA_BLA','alex-N550JK','BLA_BLA_BLA@alex-N550JK'}) at <0.778.0> exit with reason reached_max_restart_intensity in context shutdown
2015-08-31 00:14:26.298 [info] <0.385.0> Application ibot_nodes exited with reason: shutdown
2015-08-31 00:20:19.586 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-31 00:20:26.641 [warning] <0.384.0> lager_error_logger_h dropped 58 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-31 00:20:26.641 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-31 00:20:26.738 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-31 00:21:01.833 [error] <0.647.0> Supervisor ibot_nodes_sup had child 'BLA_BLA_BLA_monitor' started with ibot_nodes_srv_monitor:start_link({"BLA_BLA_BLA",'BLA_BLA_BLA','alex-N550JK','BLA_BLA_BLA@alex-N550JK'}) at <0.756.0> exit with reason normal in context child_terminated
2015-08-31 00:21:01.838 [error] <0.647.0> Supervisor ibot_nodes_sup had child 'BLA_BLA_BLA_monitor' started with ibot_nodes_srv_monitor:start_link({"BLA_BLA_BLA",'BLA_BLA_BLA','alex-N550JK','BLA_BLA_BLA@alex-N550JK'}) at <0.757.0> exit with reason normal in context child_terminated
2015-08-31 00:21:01.839 [error] <0.647.0> Supervisor ibot_nodes_sup had child 'BLA_BLA_BLA_monitor' started with ibot_nodes_srv_monitor:start_link({"BLA_BLA_BLA",'BLA_BLA_BLA','alex-N550JK','BLA_BLA_BLA@alex-N550JK'}) at <0.760.0> exit with reason normal in context child_terminated
2015-08-31 00:21:01.848 [error] <0.647.0> Supervisor ibot_nodes_sup had child 'BLA_BLA_BLA_monitor' started with ibot_nodes_srv_monitor:start_link({"BLA_BLA_BLA",'BLA_BLA_BLA','alex-N550JK','BLA_BLA_BLA@alex-N550JK'}) at <0.763.0> exit with reason normal in context child_terminated
2015-08-31 00:21:01.855 [error] <0.647.0> Supervisor ibot_nodes_sup had child 'BLA_BLA_BLA_monitor' started with ibot_nodes_srv_monitor:start_link({"BLA_BLA_BLA",'BLA_BLA_BLA','alex-N550JK','BLA_BLA_BLA@alex-N550JK'}) at <0.766.0> exit with reason normal in context child_terminated
2015-08-31 00:21:01.857 [error] <0.647.0> Supervisor ibot_nodes_sup had child 'BLA_BLA_BLA_monitor' started with ibot_nodes_srv_monitor:start_link({"BLA_BLA_BLA",'BLA_BLA_BLA','alex-N550JK','BLA_BLA_BLA@alex-N550JK'}) at <0.769.0> exit with reason normal in context child_terminated
2015-08-31 00:21:01.857 [error] <0.647.0> Supervisor ibot_nodes_sup had child 'BLA_BLA_BLA_monitor' started with ibot_nodes_srv_monitor:start_link({"BLA_BLA_BLA",'BLA_BLA_BLA','alex-N550JK','BLA_BLA_BLA@alex-N550JK'}) at <0.769.0> exit with reason reached_max_restart_intensity in context shutdown
2015-08-31 00:21:01.858 [info] <0.385.0> Application ibot_nodes exited with reason: shutdown
2015-08-31 00:26:55.635 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-31 00:27:02.692 [warning] <0.384.0> lager_error_logger_h dropped 60 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-31 00:27:02.692 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-31 00:27:02.834 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-31 00:35:24.878 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-31 00:35:31.914 [warning] <0.384.0> lager_error_logger_h dropped 58 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-31 00:35:31.914 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-31 00:35:31.998 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-31 00:40:28.195 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-31 00:40:35.214 [warning] <0.384.0> lager_error_logger_h dropped 60 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-31 00:40:35.214 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-31 00:40:35.289 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-31 00:41:40.014 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-31 00:41:47.053 [warning] <0.384.0> lager_error_logger_h dropped 61 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-31 00:41:47.054 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-31 00:41:47.146 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-31 00:44:50.780 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-31 00:44:57.835 [warning] <0.384.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-31 00:44:57.835 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-31 00:44:57.917 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-31 00:46:04.446 [error] <0.647.0> Supervisor ibot_nodes_sup had child 'BLA_BLA_BLA_monitor' started with ibot_nodes_srv_monitor:start_link({"BLA_BLA_BLA",'BLA_BLA_BLA','alex-N550JK','BLA_BLA_BLA@alex-N550JK'}) at <0.757.0> exit with reason normal in context child_terminated
2015-08-31 00:47:09.002 [error] <0.647.0> Supervisor ibot_nodes_sup had child 'BLA_BLA_BLA_monitor' started with ibot_nodes_srv_monitor:start_link({"BLA_BLA_BLA",'BLA_BLA_BLA','alex-N550JK','BLA_BLA_BLA@alex-N550JK'}) at <0.760.0> exit with reason normal in context child_terminated
2015-08-31 00:47:38.329 [error] <0.647.0> Supervisor ibot_nodes_sup had child 'BLA_BLA_BLA_monitor' started with ibot_nodes_srv_monitor:start_link({"BLA_BLA_BLA",'BLA_BLA_BLA','alex-N550JK','BLA_BLA_BLA@alex-N550JK'}) at <0.763.0> exit with reason normal in context child_terminated
2015-08-31 00:47:38.332 [error] <0.647.0> Supervisor ibot_nodes_sup had child 'BLA_BLA_BLA_monitor' started with ibot_nodes_srv_monitor:start_link({"BLA_BLA_BLA",'BLA_BLA_BLA','alex-N550JK','BLA_BLA_BLA@alex-N550JK'}) at <0.764.0> exit with reason normal in context child_terminated
2015-08-31 00:47:38.334 [error] <0.647.0> Supervisor ibot_nodes_sup had child 'BLA_BLA_BLA_monitor' started with ibot_nodes_srv_monitor:start_link({"BLA_BLA_BLA",'BLA_BLA_BLA','alex-N550JK','BLA_BLA_BLA@alex-N550JK'}) at <0.767.0> exit with reason normal in context child_terminated
2015-08-31 00:47:38.342 [error] <0.647.0> Supervisor ibot_nodes_sup had child 'BLA_BLA_BLA_monitor' started with ibot_nodes_srv_monitor:start_link({"BLA_BLA_BLA",'BLA_BLA_BLA','alex-N550JK','BLA_BLA_BLA@alex-N550JK'}) at <0.770.0> exit with reason normal in context child_terminated
2015-08-31 00:47:38.345 [error] <0.647.0> Supervisor ibot_nodes_sup had child 'BLA_BLA_BLA_monitor' started with ibot_nodes_srv_monitor:start_link({"BLA_BLA_BLA",'BLA_BLA_BLA','alex-N550JK','BLA_BLA_BLA@alex-N550JK'}) at <0.773.0> exit with reason normal in context child_terminated
2015-08-31 00:47:38.348 [error] <0.647.0> Supervisor ibot_nodes_sup had child 'BLA_BLA_BLA_monitor' started with ibot_nodes_srv_monitor:start_link({"BLA_BLA_BLA",'BLA_BLA_BLA','alex-N550JK','BLA_BLA_BLA@alex-N550JK'}) at <0.776.0> exit with reason normal in context child_terminated
2015-08-31 00:47:38.348 [error] <0.647.0> Supervisor ibot_nodes_sup had child 'BLA_BLA_BLA_monitor' started with ibot_nodes_srv_monitor:start_link({"BLA_BLA_BLA",'BLA_BLA_BLA','alex-N550JK','BLA_BLA_BLA@alex-N550JK'}) at <0.776.0> exit with reason reached_max_restart_intensity in context shutdown
2015-08-31 00:47:38.350 [info] <0.385.0> Application ibot_nodes exited with reason: shutdown
2015-08-31 00:49:21.331 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-31 00:49:28.381 [warning] <0.384.0> lager_error_logger_h dropped 60 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-31 00:49:28.381 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-31 00:49:28.467 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-31 01:00:07.776 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-31 01:00:14.852 [warning] <0.384.0> lager_error_logger_h dropped 57 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-31 01:00:14.852 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-31 01:00:14.945 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-31 01:01:59.957 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-31 01:02:00.031 [warning] <0.384.0> lager_error_logger_h dropped 63 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-31 01:02:00.032 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-31 01:02:07.008 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-31 01:02:07.090 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-31 01:05:14.943 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-31 01:05:15.031 [warning] <0.384.0> lager_error_logger_h dropped 58 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-31 01:05:15.032 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-31 01:05:21.987 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-31 01:05:22.069 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-31 01:05:45.482 [error] <0.648.0> Supervisor ibot_nodes_sup had child 'BLA_BLA_BLA_monitor' started with ibot_nodes_srv_monitor:start_link({"BLA_BLA_BLA",'BLA_BLA_BLA','alex-N550JK','BLA_BLA_BLA@alex-N550JK'}) at <0.757.0> exit with reason normal in context child_terminated
2015-08-31 01:05:50.801 [error] <0.759.0> gen_server 'BLA_BLA_BLA_monitor' terminated with reason: {timeout,{gen_server,call,[ibot_nodes_srv_connector,{restart_node,'BLA_BLA_BLA'}]}} in gen_server:call/2 line 182
2015-08-31 01:05:50.801 [error] <0.759.0> CRASH REPORT Process 'BLA_BLA_BLA_monitor' with 0 neighbours exited with reason: {timeout,{gen_server,call,[ibot_nodes_srv_connector,{restart_node,'BLA_BLA_BLA'}]}} in gen_server:terminate/7 line 804
2015-08-31 01:05:50.802 [error] <0.648.0> Supervisor ibot_nodes_sup had child 'BLA_BLA_BLA_monitor' started with ibot_nodes_srv_monitor:start_link({"BLA_BLA_BLA",'BLA_BLA_BLA','alex-N550JK','BLA_BLA_BLA@alex-N550JK'}) at <0.759.0> exit with reason {timeout,{gen_server,call,[ibot_nodes_srv_connector,{restart_node,'BLA_BLA_BLA'}]}} in context shutdown_error
2015-08-31 01:06:11.135 [error] <0.648.0> Supervisor ibot_nodes_sup had child 'BLA_BLA_BLA_monitor' started with ibot_nodes_srv_monitor:start_link({"BLA_BLA_BLA",'BLA_BLA_BLA','alex-N550JK','BLA_BLA_BLA@alex-N550JK'}) at <0.762.0> exit with reason normal in context child_terminated
2015-08-31 01:06:16.453 [error] <0.764.0> gen_server 'BLA_BLA_BLA_monitor' terminated with reason: {timeout,{gen_server,call,[ibot_nodes_srv_connector,{restart_node,'BLA_BLA_BLA'}]}} in gen_server:call/2 line 182
2015-08-31 01:06:16.454 [error] <0.764.0> CRASH REPORT Process 'BLA_BLA_BLA_monitor' with 0 neighbours exited with reason: {timeout,{gen_server,call,[ibot_nodes_srv_connector,{restart_node,'BLA_BLA_BLA'}]}} in gen_server:terminate/7 line 804
2015-08-31 01:06:16.454 [error] <0.648.0> Supervisor ibot_nodes_sup had child 'BLA_BLA_BLA_monitor' started with ibot_nodes_srv_monitor:start_link({"BLA_BLA_BLA",'BLA_BLA_BLA','alex-N550JK','BLA_BLA_BLA@alex-N550JK'}) at <0.764.0> exit with reason {timeout,{gen_server,call,[ibot_nodes_srv_connector,{restart_node,'BLA_BLA_BLA'}]}} in context shutdown_error
2015-08-31 01:08:04.185 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-31 01:08:11.236 [warning] <0.384.0> lager_error_logger_h dropped 57 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-31 01:08:11.236 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-31 01:08:11.318 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-31 01:11:14.504 [error] <0.782.0> gen_server 'BLA_BLA_BLA_monitor' terminated with reason: {timeout,{gen_server,call,[ibot_nodes_srv_connector,{restart_node,'BLA_BLA_BLA'}]}} in gen_server:call/2 line 182
2015-08-31 01:11:14.504 [error] <0.782.0> CRASH REPORT Process 'BLA_BLA_BLA_monitor' with 0 neighbours exited with reason: {timeout,{gen_server,call,[ibot_nodes_srv_connector,{restart_node,'BLA_BLA_BLA'}]}} in gen_server:terminate/7 line 804
2015-08-31 01:11:14.505 [error] <0.648.0> Supervisor ibot_nodes_sup had child 'BLA_BLA_BLA_monitor' started with ibot_nodes_srv_monitor:start_link({"BLA_BLA_BLA",'BLA_BLA_BLA','alex-N550JK','BLA_BLA_BLA@alex-N550JK'}) at <0.782.0> exit with reason {timeout,{gen_server,call,[ibot_nodes_srv_connector,{restart_node,'BLA_BLA_BLA'}]}} in context shutdown_error
2015-08-31 01:12:04.426 [error] <0.648.0> Supervisor ibot_nodes_sup had child 'BLA_BLA_BLA_monitor' started with ibot_nodes_srv_monitor:start_link({"BLA_BLA_BLA",'BLA_BLA_BLA','alex-N550JK','BLA_BLA_BLA@alex-N550JK'}) at <0.821.0> exit with reason killed in context shutdown_error
2015-08-31 01:15:43.730 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-31 01:15:50.966 [warning] <0.384.0> lager_error_logger_h dropped 61 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-31 01:15:50.966 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-31 01:15:51.051 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-31 01:17:00.620 [error] <0.762.0> gen_server 'BLA_BLA_BLA_monitor' terminated with reason: {timeout,{gen_server,call,[ibot_nodes_srv_connector,{restart_node,'BLA_BLA_BLA'}]}} in gen_server:call/2 line 182
2015-08-31 01:17:00.620 [error] <0.762.0> CRASH REPORT Process 'BLA_BLA_BLA_monitor' with 0 neighbours exited with reason: {timeout,{gen_server,call,[ibot_nodes_srv_connector,{restart_node,'BLA_BLA_BLA'}]}} in gen_server:terminate/7 line 804
2015-08-31 01:17:00.620 [error] <0.647.0> Supervisor ibot_nodes_sup had child 'BLA_BLA_BLA_monitor' started with ibot_nodes_srv_monitor:start_link({"BLA_BLA_BLA",'BLA_BLA_BLA','alex-N550JK','BLA_BLA_BLA@alex-N550JK'}) at <0.762.0> exit with reason {timeout,{gen_server,call,[ibot_nodes_srv_connector,{restart_node,'BLA_BLA_BLA'}]}} in context shutdown_error
2015-08-31 01:25:47.848 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-31 01:25:54.898 [warning] <0.384.0> lager_error_logger_h dropped 63 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-31 01:25:54.898 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-31 01:25:54.981 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-31 01:26:46.229 [error] <0.647.0> Supervisor ibot_nodes_sup had child 'BLA_BLA_BLA_monitor' started with ibot_nodes_srv_monitor:start_link({"BLA_BLA_BLA",'BLA_BLA_BLA','alex-N550JK','BLA_BLA_BLA@alex-N550JK'}) at <0.762.0> exit with reason killed in context shutdown_error
2015-08-31 01:27:47.441 [error] <0.765.0> gen_server 'BLA_BLA_BLA_monitor' terminated with reason: {timeout,{gen_server,call,[ibot_nodes_srv_connector,{restart_node,'BLA_BLA_BLA'}]}} in gen_server:call/2 line 182
2015-08-31 01:27:47.441 [error] <0.765.0> CRASH REPORT Process 'BLA_BLA_BLA_monitor' with 0 neighbours exited with reason: {timeout,{gen_server,call,[ibot_nodes_srv_connector,{restart_node,'BLA_BLA_BLA'}]}} in gen_server:terminate/7 line 804
2015-08-31 01:27:47.441 [error] <0.647.0> Supervisor ibot_nodes_sup had child 'BLA_BLA_BLA_monitor' started with ibot_nodes_srv_monitor:start_link({"BLA_BLA_BLA",'BLA_BLA_BLA','alex-N550JK','BLA_BLA_BLA@alex-N550JK'}) at <0.765.0> exit with reason {timeout,{gen_server,call,[ibot_nodes_srv_connector,{restart_node,'BLA_BLA_BLA'}]}} in context shutdown_error
2015-08-31 01:41:42.995 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-31 01:41:43.527 [warning] <0.384.0> lager_error_logger_h dropped 58 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-31 01:41:43.528 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-31 01:41:50.236 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-31 01:41:50.683 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-31 01:45:13.470 [error] <0.647.0> Supervisor ibot_nodes_sup had child 'BLA_BLA_BLA_monitor' started with ibot_nodes_srv_monitor:start_link({"BLA_BLA_BLA",'BLA_BLA_BLA','alex-N550JK','BLA_BLA_BLA@alex-N550JK'}) at <0.772.0> exit with reason killed in context shutdown_error
2015-08-31 01:47:10.026 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-31 01:47:17.068 [warning] <0.384.0> lager_error_logger_h dropped 61 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-31 01:47:17.068 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-31 01:47:17.151 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-31 01:47:56.324 [error] <0.756.0> gen_server 'BLA_BLA_BLA_monitor' terminated with reason: {timeout,{gen_server,call,[ibot_nodes_srv_connector,{restart_node,'BLA_BLA_BLA'}]}} in gen_server:call/2 line 182
2015-08-31 01:47:56.324 [error] <0.756.0> CRASH REPORT Process 'BLA_BLA_BLA_monitor' with 0 neighbours exited with reason: {timeout,{gen_server,call,[ibot_nodes_srv_connector,{restart_node,'BLA_BLA_BLA'}]}} in gen_server:terminate/7 line 804
2015-08-31 01:47:56.324 [error] <0.647.0> Supervisor ibot_nodes_sup had child 'BLA_BLA_BLA_monitor' started with ibot_nodes_srv_monitor:start_link({"BLA_BLA_BLA",'BLA_BLA_BLA','alex-N550JK','BLA_BLA_BLA@alex-N550JK'}) at <0.756.0> exit with reason {timeout,{gen_server,call,[ibot_nodes_srv_connector,{restart_node,'BLA_BLA_BLA'}]}} in context shutdown_error
2015-08-31 01:50:48.328 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-31 01:50:55.383 [warning] <0.384.0> lager_error_logger_h dropped 61 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-31 01:50:55.383 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-31 01:50:55.463 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-31 01:51:38.114 [error] <0.647.0> Supervisor ibot_nodes_sup had child 'BLA_BLA_BLA_monitor' started with ibot_nodes_srv_monitor:start_link({"BLA_BLA_BLA",'BLA_BLA_BLA','alex-N550JK','BLA_BLA_BLA@alex-N550JK'}) at <0.760.0> exit with reason normal in context child_terminated
2015-08-31 01:51:38.114 [error] <0.647.0> Supervisor ibot_nodes_sup had child 'BLA_BLA_BLA_monitor' started with ibot_nodes_srv_monitor:start_link({"BLA_BLA_BLA",'BLA_BLA_BLA','alex-N550JK','BLA_BLA_BLA@alex-N550JK'}) at <0.761.0> exit with reason normal in context shutdown_error
2015-08-31 01:53:49.726 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-31 01:53:56.802 [warning] <0.384.0> lager_error_logger_h dropped 63 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-31 01:53:56.802 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-31 01:53:56.883 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-31 01:54:31.088 [error] <0.757.0> gen_server 'BLA_BLA_BLA_monitor' terminated with reason: bad return value: {stop,normal,shutdown_ok,{state,"BLA_BLA_BLA",'BLA_BLA_BLA','alex-N550JK','BLA_BLA_BLA@alex-N550JK'}}
2015-08-31 01:54:31.088 [error] <0.757.0> CRASH REPORT Process 'BLA_BLA_BLA_monitor' with 0 neighbours exited with reason: bad return value: {stop,normal,shutdown_ok,{state,"BLA_BLA_BLA",'BLA_BLA_BLA','alex-N550JK','BLA_BLA_BLA@alex-N550JK'}} in gen_server:terminate/7 line 804
2015-08-31 01:54:31.089 [error] <0.648.0> Supervisor ibot_nodes_sup had child 'BLA_BLA_BLA_monitor' started with ibot_nodes_srv_monitor:start_link({"BLA_BLA_BLA",'BLA_BLA_BLA','alex-N550JK','BLA_BLA_BLA@alex-N550JK'}) at <0.757.0> exit with reason bad return value: {stop,normal,shutdown_ok,{state,"BLA_BLA_BLA",'BLA_BLA_BLA','alex-N550JK','BLA_BLA_BLA@alex-N550JK'}} in context child_terminated
2015-08-31 01:54:31.092 [error] <0.758.0> gen_server 'BLA_BLA_BLA_monitor' terminated with reason: bad return value: {stop,normal,shutdown_ok,{state,"BLA_BLA_BLA",'BLA_BLA_BLA','alex-N550JK','BLA_BLA_BLA@alex-N550JK'}}
2015-08-31 01:54:31.092 [error] <0.758.0> CRASH REPORT Process 'BLA_BLA_BLA_monitor' with 0 neighbours exited with reason: bad return value: {stop,normal,shutdown_ok,{state,"BLA_BLA_BLA",'BLA_BLA_BLA','alex-N550JK','BLA_BLA_BLA@alex-N550JK'}} in gen_server:terminate/7 line 804
2015-08-31 01:54:31.093 [error] <0.648.0> Supervisor ibot_nodes_sup had child 'BLA_BLA_BLA_monitor' started with ibot_nodes_srv_monitor:start_link({"BLA_BLA_BLA",'BLA_BLA_BLA','alex-N550JK','BLA_BLA_BLA@alex-N550JK'}) at <0.758.0> exit with reason bad return value: {stop,normal,shutdown_ok,{state,"BLA_BLA_BLA",'BLA_BLA_BLA','alex-N550JK','BLA_BLA_BLA@alex-N550JK'}} in context shutdown_error
2015-08-31 01:57:08.530 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-31 01:57:15.560 [warning] <0.384.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-31 01:57:15.560 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-31 01:57:15.645 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
2015-08-31 01:57:42.280 [error] <0.648.0> Supervisor ibot_nodes_sup had child 'BLA_BLA_BLA_monitor' started with ibot_nodes_srv_monitor:start_link({"BLA_BLA_BLA",'BLA_BLA_BLA','alex-N550JK','BLA_BLA_BLA@alex-N550JK'}) at <0.757.0> exit with reason killed in context shutdown_error
2015-08-31 02:00:21.619 [info] <0.385.0> Application lager started on node 'core@alex-N550JK'
2015-08-31 02:00:28.642 [warning] <0.384.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2015-08-31 02:00:28.642 [info] <0.385.0> Application mnesia exited with reason: stopped
2015-08-31 02:00:28.769 [info] <0.385.0> Application mnesia started on node 'core@alex-N550JK'
